flask_accept/__init__.py

```diff
@@ -36,7 +37,12 @@ def __call__(self, *args, **kwargs):
         raise NotAcceptable(description)
 
     def __get__(self, instance, owner):
-        return partial(self.__call__, instance)
+        func = partial(self.__call__, instance)
+
+        # flask-restplus use doc string to swagger docment
+        func.__doc__ = self.__func_doc_str
```

Comment:
Could you make this something like:

```python
func.__doc__ = self.fallback.__doc__
```

We don't actually need to store this on the instance. I'm also assuming Swagger can handle the case where the docstring is false-y so we don't need the `if func.__doc__ else ""` but feel free to correct me if that's wrong.

Reply:
you are correct :)